src/shared/components/organisms/navbar/navbar.constants.ts

```diff
@@ -36,11 +36,11 @@ export const loggedOutNavList: INavbarItem[] = [
     url: routes.home,
   },
   {
-    name: 'About us',
```

Comment:
are these removed from the design also? Make sure these are also removed from design